spec/takeaway_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'takeaway'
+
+describe Takeaway do
+
+
+  it { is_expected.to respond_to :check_menu }
+
+  it { is_expected.to respond_to(:add_to_order).with(2).arguments }
+
+  it 'should be initialised with a default menu' do
+    expect(subject.menu).to eq Takeaway::MENU
+  end
+
+    describe 'check' do
+
+      it 'should print out the list of dishes and prices' do
+        expect(subject.check_menu).to eq Takeaway::MENU
+      end
+
+    end
+
+    describe 'add_to_order' do
+
+      it 'should raise an error message if the dish does not match an item in the menu' do
+        expect{subject.add_to_order(1, :pizza)}.to raise_error "Sorry! That's not on the menu!"
```

Comment:
Line is too long. [95/80]<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.